src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarPrivateArtwork.tsx

```diff
@@ -35,8 +31,7 @@ export const ArtworkSidebarPrivateArtwork: React.FC<ArtworkSidebarPrivateArtwork
     artwork
   )
 
-  const isPrivateArtwork =
-    privateArtworksEnabled && data.visibilityLevel === "UNLISTED"
+  const isPrivateArtwork = data.visibilityLevel === "UNLISTED"
```

Comment:
Is it possible to change both instances of referencing the magic string to instead use `artwork?.isUnlisted`? Thinking in the future when we potentially will have multiple "private" types which we want to hide

Reply:
Good call, Ill do a clean up pass as part of this PR